app/src/main/kotlin/com/softteco/template/ui/components/PrimaryButton.kt

```diff
@@ -0,0 +1,57 @@
+package com.softteco.template.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.softteco.template.ui.theme.Dimens
+
+@Composable
+fun PrimaryButton(
+    buttonText: String,
+    showLoader: Boolean,
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit,
+) {
+    Box(
```

Comment:
It looks redundant. Paddings could be different for different screens.

Reply:
We shouldn't set paddings for common button, it should come from modifier parameter and should be changable.